SU-SWS/stanford_basic
# D8CORE-635: Style Tweaks

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-635_

# Needed By (Date)
- Sprint close


# Urgency
- Styling fixes

# Steps to Test

- Checkout this branch
- Navigate to any page with an H1 
- Verify you see the larger style

- Create or find a quote through the WYSIWYG
- Verify it has larger font

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-635

## Related PRs

## More Information

## Folks to notify
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
